api/http.go

```diff
@@ -62,8 +62,12 @@ func NewCatalystAPIRouter(cli config.Cli, vodEngine *pipeline.Coordinator, bal b
 		Config:   cli,
 		Lapi:     lapi,
 	}
+	analyticsApiHandlers := &handlers.AnalyticsHandlersCollection{}
```

Comment:
maybe make a constructor function for this as I often find we end up needing some logic when setting up a struct like this and it can be a pain in the future updating everywhere.

Reply:
Updated in 966e638a1cc48551294ffc960a8f82364098cb9c